docs/pages/2020_Intro_Week/sections/workflows_adv.rst

```diff
@@ -59,8 +71,8 @@ In the :py:meth:`~aiida.workflows.arithmetic.multiply_add.MultiplyAddWorkChain.v
 However, there is one thing that we didn't consider here: what if the calculation failed to produce the sum?
 For such a trivial example of a calculation adding two numbers, although possible, it is very unlikely that it will fail.
 Nonetheless, if it *were* to fail, the work chain would except because the line ``self.ctx.addition.outputs.sum`` will raise an ``AttributeError``.
-In this case, where the work chain just runs a single calculation, that is not such a big deal, but for real-life work chains, that run a number of calculations in sequence, having the work chain except will cause all the work up to that point to be lost.
-Take as an example a workflow that computes the phonons of a crystal structure, when using Quantum ESPRESSO, that takes a sequence of four calculations:
+In this case, where the work chain just runs a single calculation that is not such a big deal, but for real-life work chains that run a number of calculations in sequence, having the work chain except will cause all the work up to that point to be lost.
+Take as an example a workflow that computes the phonons of a crystal structure using Quantum ESPRESSO over a sequence of four calculations:
```

Comment:
Sure, but personally (and probably others reading through) don't know or care that for QE it is specifically 4. So I twisted the sentence instead so that the workflow represents how it _could_ be done using QE as the example code of choice. But Maybe I can find a way to combine the two meanings if it's important?

Reply:
You could simplify it even more to "Take as an example a workflow that computes the phonons of a crystal structure using Quantum ESPRESSO"